### DISCUSSION TITLE
How do I directly obtain the CPU and GPU temperatures on Windows? I have an Intel NUC A770M.

### DISCUSSION THREAD

**Discussion by byclear** [opcm/pcm#827] [Q&A]
I just want to do it simply via the command line, as I am not a professional C developer and I don’t know how to compile. Is there a ready-made .exe file that I can just run to get the temperatures? Can someone assist me, please?

**Comment by rdementi** [comment]
I think you can try to use these methods as a non-developer: https://stackoverflow.com/questions/39738494/get-cpu-temperature-in-cmd-power-shell

**Comment by byclear** [comment]
> 我认为您可以尝试以非开发人员的身份使用这些方法：https://stackoverflow.com/questions/39738494/get-cpu-temperature-in-cmd-power-shell

I was trying to get it via python manipulating terminal commands, and the page you provided doesn't address how to get the graphics card temperature for the Intel A770M. I was directed to that PCM project from Intel's expired monitoring tool. But since there is no .exe for that PCM project, I need to get it via terminal or python. How should I ask for the group. I even started a help request on Intel's official website, but it doesn't give a solution. I am always told to check the temperature through task manager instead of command line form.
 @rdementi

**Comment by byclear** [comment]
https://www.intel.com/content/www/us/en/developer/articles/tool/power-gadget.html?wapkw=Intel%20Power%20Gadget

For example, this one points to PCM @rdementi

**Comment by opcm** [comment]
I believe Intel Power Gadget was providing statistics for CPU and integrated graphics which are on the same package. This means package temperature is same or close to CPU or integrated GT graphics cores. But you have a discrete Intel Arc A770M graphics card and the tooling is different. You might want to ask in this Intel Arc graphics forum: https://community.intel.com/t5/Intel-ARC-Graphics/bd-p/arc-graphics

**Comment by byclear** [comment]
Thanks, I'll try to solve it
